generators/generator-openapi/src/main/java/org/eclipse/vorto/codegen/openapi/templates/OpenAPITemplate.xtend

```diff
@@ -373,6 +482,30 @@ class OpenAPITemplate implements IFileTemplate<InformationModel> {
 		      - status
 		      - error
 		      - message
+		    Thing:
+		      type: object
+		      required:
+		      - thingId
+		      - policyId
+		      - attributes
+		      - features
+		      properties:
+		        thingId:
+		          type: string
+		          description: Unique identifier representing the thing
+		        policyId:
+		          type: string
+		          description: The policy ID used for controlling access to this thing, managed by resource `/policies/{policyId}`
+		        attributes:
+		          $ref: '#/components/schemas/Attributes'
+		        features:
+		          $ref: '#/components/schemas/Features'
```

Comment:
Where is Features defined ? Shouldn't this be the «infomodel.name»Features ?

Reply:
Attributes is defined in line 503.

Reply:
I will add FeatureDefinition and FeatureProperties in a next step. to be in line with the IoT Things schema.

Reply:
Hi @philbuettner 
AFAIK the Features specifically for the information model are already generated. In this case the Thing model would directly reference that one. The one you said from existing Things Schema is a generic structure , and not strong typed to the function blocks.

Reply:
Ok, what should be the content of /things/{thingId} then?

Reply:
the same Thing model as you have specified except that the **features** references #schema/components/«infomodel.name»Features